CONTRIBUTING.md

```diff
@@ -0,0 +1,107 @@
+# Contributing guidelines
+
+Thank you for you contribution. It's really appreciated.
+
+Set of guidelines follows. Use your best judgment, the world is not black and white and these guidelines are not hard rules.
+
+**Table of contents**
+
+* [How can I contribute?](#how-can-i-contribute)
+    * [Reporting bugs](#reporting-bugs)
+* [Issue labels](#issue-labels)
+
+## How can I contribute?
+
+### Reporting bugs
+
+#### Before submitting a bug report
+
+* **Check** the [forum][pythonista-forum] for a list of common issues and possible workarounds.
+* **Check** wheter the issue you are experiencing is [already reported][pythonista-issues]. If it is already reported, you can click on the _Subscribe_ button to receive further development info or you can add more details.
+* **Check** if you're using the [latest Pythonista version][pythonista-app-store]. Bug reports should be based on the latest available version. If you're using Pythonista beta version, test your issue with the latest available version from the Test Flight.
+
+#### How do I submit a bug report?
+
+* **Isolate problem** - The first step is to identify what the problem is. Saying "something is wrong" is not helpful. Saying what exactly is wrong, how to reproduce it, is.
+* **Separate issues** - If you have multiple issues, please, file them separately.
+* **Use clear and descriptive title** - title like "help system" is not helpful.
+* **Steps to reproduce** - Include all steps how to reproduce your issue. More details, faster resolution. Don't just say what you did, but explain how you did it (tapping on, hw keyboard, ...).
+* **Expected behavior** - If something doesn't work as expected, include what exactly should happen.
+* **Sample code** - Sample code demonstrating your issue is very helpful. If it's short, include it directly in the issue. Read [Quoting code][github-quoting-code] to learn how to include code in the issue. [Gist][github-gist] is preferred in case your code is too long.
+* **System information** - It's always good to attach system information.
+* **Ask yourself** - Is my description detailed enough that everyone understands what's my problem?
+
+#### System information
+
+Here's an example of system information to attach (preformatted Markdown):
+
+```
+**System Information**
+
+* Pythonista 3.1.1 (311015), Default interpreter 3.6.1
+* iOS 11.0.3, model iPad, resolution (portrait) 2048.0 x 2732.0 @ 2.0
+```
+
+If you don't know from where to get this information, download and run [sysinfo.py][pythonista-sysinfo-script] script.
+
+Why? Some bugs may appear on specific device, specific iOS version and it's very helpful to know more about your system. Problem can be identified and fixed more quickly.
+
+If you feel bad about sharing system information of your device, don't attach it. It's mandatory, but it's really helpful.
```

Comment:
That should probably be "It's *not* mandatory".

Reply:
Fixed, thanks for catching this!